Package.swift

```diff
@@ -0,0 +1,86 @@
+// swift-tools-version:5.4.0
+
+import PackageDescription
+
+let package = Package(
+    name: "Ably",
+    platforms: [
+        .macOS(.v10_11),
+        .iOS(.v9),
+        .tvOS(.v10)
+    ],
+    products: [
+        .library(
+            name: "Ably",
+            targets: ["Ably"]
+        ),
+    ],
+    dependencies: [
+        .package(name: "msgpack", url: "https://github.com/rvi/msgpack-objective-C", from: "0.4.0"),
+        .package(name: "AblyDeltaCodec", url: "https://github.com/ably/delta-codec-cocoa", .branch("main"))
```

Comment:
Instead of relying on `main`, does it make sense to use:
```.package(name: "AblyDeltaCodec", url: "https://github.com/ably/delta-codec-cocoa/blob/main/Package.swift")```, **without** the version?
or 
```.package(name: "AblyDeltaCodec", url: "https://github.com/ably/delta-codec-cocoa/blob/main/Package.swift", from: "1.3.0")``` or explicitly specifying the version/ git tag

Reply:
definitely, this should be a specific version

Reply:
done

Reply:
35a38b82f09937daf2c63cb7248f5319ab628b17